### PR TITLE
Remove last nibabel occurences

### PR DESCRIPTION
**Pull request by AlexandreAbraham**

### CODE REVIEW

File: examples/02_decoding/plot_haxby_searchlight.py [hunk 1]
```diff
@@ -23,7 +22,7 @@
 print('Functional nifti image (4D) is located at: %s' % haxby_dataset.func[0])
 
 fmri_filename = haxby_dataset.func[0]
-fmri_img = nibabel.load(fmri_filename)
+fmri_img = load_img(fmri_filename)
```
Comment (GaelVaroquaux) [inline examples/02_decoding/plot_haxby_searchlight.py]: Do we actually need to load it in memory? It seems to me that we do not.

**Comment by GaelVaroquaux** [comment]
LGTM, aside my remark that we shouldn't call load_img on fmri_filename, but pass it directly to index_img lower in the script.

File: examples/02_decoding/plot_haxby_searchlight.py [hunk 2]
```diff
@@ -23,7 +22,6 @@
 print('Functional nifti image (4D) is located at: %s' % haxby_dataset.func[0])
 
 fmri_filename = haxby_dataset.func[0]
-fmri_img = nibabel.load(fmri_filename)
```
Comment (GaelVaroquaux) [inline examples/02_decoding/plot_haxby_searchlight.py]: I don't think that the example runs, now: you need to change 'fmri_img' to 'fmri_filename' below (on line 33).

Comment (AlexandreAbraham) [inline examples/02_decoding/plot_haxby_searchlight.py]: Yes, I've noticed it a bit too late.

**Pull request merged**